server/store.go

```diff
@@ -415,3 +422,34 @@ func (p *Plugin) getReminderPreference(userID string) bool {
 
 	return preference
 }
+
+func (p *Plugin) saveAllowIncomingTaskRequestsPreference(userID string, preference bool) error {
+	preferenceString := strconv.FormatBool(preference)
+	appErr := p.API.KVSet(allowIncomingTaskRequestsKey(userID), []byte(preferenceString))
+	if appErr != nil {
+		return appErr
+	}
+	return nil
+}
+
+// getAllowIncomingTaskRequestsPreference - gets user preference on allowing incoming task requests from other users - default value will be true if in case any error
+func (p *Plugin) getAllowIncomingTaskRequestsPreference(userID string) bool {
+	preferenceByte, appErr := p.API.KVGet(allowIncomingTaskRequestsKey(userID))
+	if appErr != nil {
+		p.API.LogError("error getting the allow incoming task requests preference, err=", appErr.Error())
```

Comment:
Nit pick: It's idiomatic to just return the error (Maybe warped) and let the caller deal with it.